### ISSUE TITLE
HDD-OSD-Icons-Pack internet error

### ISSUE THREAD

**Issue by ahmedmoselhi** [GDX-X/PFS-BatchKit-Manager#27]
when i gtry to install apps or utilites on hdd i got this error

```
Checking internet Or Website connection... For HDD-OSD ART
Unable to connect to internet Or Website
You will switch to offline mode
```
my internet connection is working well though..

**Comment by GDX-X** [comment]
this does not necessarily mean internet, but access to the archive.org site may not be available at the moment or while the site is blocking your internet connection

**Comment by ahmedmoselhi** [comment]
```
wget " -q --show-progress https://archive.org/download/OPLM_ART_2022_07/OPLM_ART_2022_07.zip/PS1%%2FSCES_000.01%%2FSCES_000.01_COV.jpg -O "%~dp0TMP\SCES_000.01_COV.jpg" >nul 2>&1
 -q --show-progress https://archive.org/download/OPLM_ART_2022_07/OPLM_ART_2022_07.zip/PS1%%2FSCES_000.01%%2FSCES_000.01_COV.jpg -O %~dp0TMP\SCES_000.01_COV.jpg >nul 2>&1: Scheme missing.
```

**Comment by ahmedmoselhi** [comment]
wget command throws an error

**Comment by GDX-X** [comment]
Fixed in 1.1.3
